demo/generator.sh

```diff
@@ -4,5 +4,5 @@ SERVER=localhost
 PORT=2003
 while true; do
 	R=$((RANDOM % 100))
-	echo my.awesome.path $R `date +%s` | nc ${SERVER} ${PORT} -w 1
+	echo "my.awesome.path1;b=123;a=1;c=5" $R `date +%s` | nc ${SERVER} ${PORT} -w 1
```

Comment:
so the main problem is that we need to change metrics format everywhere. This can be kind of problematic.

Reply:
If metrics that we want to add are graphite-without-tags + influx-like templates, we can parse it to this representation (or even add this logic to carbon-clickhouse directly).